src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import duke.util.TaskList;
+import duke.util.Ui;
+import duke.util.Parser;
+import duke.util.Storage;
+
+import java.io.IOException;
+
+/**
+ * The Duke class represents the TaskMate bot that allows users to
+ * manage tasks by adding, deleting, marking and un-marking tasks.
+ *
+ * @author Win Sheng
+ * @since 3 September 2023
+ */
```

Comment:
I like how you updated the java docs to include date updated 😃 
This was done for all documents which is consistent!